### PR TITLE
Adding trodi 

### PR DESCRIPTION
**Pull request by scottstanie**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/trodi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  entry_points:
+    - trodi = trodi.cli:average_and_label
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+    - "numpy ==1.19.*"
```
Comment (dopplershift) [inline recipes/trodi/meta.yaml:25]: The `setup.py` has these as e.g. `numpy >=1.16.0`. Any reason these need to be pinned tighter here?

Comment (scottstanie) [reply recipes/trodi/meta.yaml]: no I don't think so, had that matched to my current environment, but probably not necessary

File: recipes/trodi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+    - "numpy =>1.16.*"
+    - "xarray ==0.16.*"
+    - "netCDF4 ==1.5.*"
```
Comment (dopplershift) [inline recipes/trodi/meta.yaml:27]: Sorry, what I meant was that this set then should just be then:
```yaml
- numpy >=1.16.0
- xarray >=0.16.1
- netcdf4 >=1.5.4
```

Comment (scottstanie) [reply recipes/trodi/meta.yaml]: ok gotcha. Made that change.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**